FinSearchUnified/Subscriber/Widgets.php

```diff
@@ -33,4 +116,25 @@ public function beforeListingCountAction(Enlight_Hook_HookArgs $args)
             }
         }
     }
+
+    /**
+     * @param Enlight_Controller_Request_RequestHttp $request
+     *
+     * @return string
+     * @throws Exception
+     */
+    private function parseReferUrl($request)
+    {
+        $referrer = isset($_SERVER['HTTP_REFERER']) ? $_SERVER['HTTP_REFERER'] : '';
```

Comment:
I do not really like using superglobals directly. How about `$request->headers->get('referer')`?

Reply:
ahan I see. Just out of curiosity, what is the difference between `getHeader` and calling it like this? because I think getHeader was returning null so I changed it to super global. I wasn't aware of this property.

Reply:
I guess this seems to be failing on travis.

Reply:
Im fine with it.